osu.Game/Graphics/UserInterface/ExpandingBar.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Shapes;
+using osuTK;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    /// <summary>
+    /// A rounded bar which can be expanded or collapsed.
+    /// Generally used for tabs or breadcrumbs.
+    /// </summary>
+    public class ExpandingBar : Circle
```

Comment:
I don't see any game-specific code here. Why this class is not in the framework?

Reply:
Because it's osu! design.